funds-manager/funds-manager-server/src/custody_client/gas_wallets.rs

```diff
@@ -31,13 +31,13 @@ impl CustodyClient {
     // | Handlers |
     // ------------
 
-    /// Refill gas for all active wallets
+    /// Refill gas for all gas wallets
     pub(crate) async fn refill_gas_for_active_wallets(
```

Comment:
nit: could the function name reflect this change as well, i.e. `refill_all_gas_wallets`

also the comment below above the `get_all_gas_wallets` call